Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -849,8 +897,29 @@ private static float GetInputSystemDefaultDeadZone()
 
         private static float GetGamepadDeadZone(GamepadSlot gamepadSlot)
         {
-            // TODO implement me!
-            return GetInputSystemDefaultDeadZone();
+            // TODO remove me when init is implemented
+            InitializeGamepadConfigs();
+
+            switch (gamepadSlot)
+            {
+                case GamepadSlot.Slot1:
+                case GamepadSlot.Slot2:
+                case GamepadSlot.Slot3:
+                case GamepadSlot.Slot4:
+                case GamepadSlot.Slot5:
+                case GamepadSlot.Slot6:
+                case GamepadSlot.Slot7:
+                case GamepadSlot.Slot8:
+                case GamepadSlot.Slot9:
+                case GamepadSlot.Slot10:
+                case GamepadSlot.Slot11:
+                case GamepadSlot.Slot12:
+                    return s_GamepadConfigs[(int)gamepadSlot].deadzone;
```

Comment:
I still do not see what we gain here by having GamepadSlot as an enum compared to plain integer? The only special thing seems to be "All" which could be a constant used as default argument? basically the enum prevents any kind of loop/enumeration processing on top which is a likely use-case?

Reply:
I think doing enums is fine for now, let's check that on first customer API feedback and if int's are preferred then switch to int's